docs/source/noise.rst

```diff
@@ -1002,7 +992,7 @@ Example 1: Rabi Sequence with Noisy Readout
 
     for jj, theta in enumerate(thetas):
         for kk, p00 in enumerate(p00s):
-            qc.qam.random_seed = hash((jj, kk))
+            qc.qam.random_seed = 1
```

Comment:
The `hash(...)` caused QVM to break in a weird way that a seed of `1` does not

Reply:
Just to confirm, the error came from the QVM server, it didn't appear to be a pyQuil issue?

Reply:
Correct. The error originated in QVM